db/migrate/20170207184733_remove_email_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveEmailFromUsers < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :users, :email, :string
```

Comment:
I removed it to avoid modifying Devise's migration. When I added Devise to User it inserted back this column.